Makefile

```diff
@@ -523,15 +495,15 @@ PUBLISH_BINTRAY_REDHAT += \
 endif
 
 ifeq ($(TARGET_PLATFORM),win32)
-electron-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
+electron-installer-portable: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH)-portable.exe
 electron-installer-nsis: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe
 cli-installer-zip: $(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
 TARGETS += \
-	electron-installer-zip \
+	electron-installer-portable \
```

Comment:
Does this change mean that the build system will stop creating Linux zip files? Perhaps we should add a `electron-installer-portable` target and not remove `electron-installer-zip` ?